example/app.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
we could delete app.json here to make the example more minimal, or remove some of these keys that aren't really needed (icon, splash, etc)